### ISSUE TITLE
Unable to establish SSL connection

### ISSUE THREAD

**Issue by Aviv19** [Junhua-Liao/Light-ASD#3]
Hello,

I try to download python "train.py --dataPathAVA AVADataPath --download"  but get this error:
ERROR: cannot verify s3.amazonaws.com's certificate, issued by `/C=US/O=Amazon/CN=Amazon RSA 2048 M01':
  Unable to locally verify the issuer's authority.
To connect to s3.amazonaws.com insecurely, use `--no-check-certificate'.
Unable to establish SSL connection.

**Comment by Junhua-Liao** [comment]
Sorry, I haven't encountered this issue before. There is a solution available online which involves running the following code.

sudo yum install -y ca-certificates

I'm not sure if it will help you resolve the issue.

**Comment by Aviv19** [comment]
Thanks, I just needed to update wget

**Comment by Aviv19** [comment]
Hi, the dataset is very large, something like 302 GB. Is there a way to download a smaller dataset to train the model?

Thanks

**Comment by Junhua-Liao** [comment]
Sorry, this is the only way that I currently know.

The original video, data generated during the preprocessing stage, and the final dataset altogether take up about 300G. The size of the final dataset is less than 100G. If you do not have enough storage space, you can delete `orig_videos`(167G) after obtaining `clips_videos`(85G), or delete `orig_audios`(44G) after obtaining `clips_audios`(6.4G).

If you only want to obtain the training set, you can set the dataType to 'train' in `utils/tools.py`.
If you only want to obtain the validation set, you can access this [repository](https://github.com/huangmin9966/FSDNet_RealTimeArch) .
